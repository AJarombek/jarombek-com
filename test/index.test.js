/**
 * Placeholder for future testing suite with Jest
 * @author Andrew Jarombek
 * @since 3/21/2018
 */

import React from 'react';
import {shallow} from 'enzyme';

test('No Test', () => {
   expect(true);
});