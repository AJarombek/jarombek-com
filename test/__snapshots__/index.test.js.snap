// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<div
  className="jarombek-main"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Andrew Jarombek
    </title>
    <meta
      content="Andrew Jarombek"
      name="author"
    />
    <meta
      content="Andrew Jarombek's Personal Website and
                            Software Development Blog"
      name="description"
    />
    <link
      href="https://jarombek.com"
      rel="canonical"
    />
    <link
      href="test-file-stub"
      rel="icon"
    />
  </HelmetWrapper>
  <div
    className="title-container"
  >
    <h1
      className="title"
    >
      Andrew Jarombek
    </h1>
  </div>
  <p
    className="sub-title"
  >
    Website Under Construction
  </p>
  <Link
    className="blog-button"
    replace={false}
    to="/blog"
  >
    <Button
      activeColor="primary"
      onClick={[Function]}
      passiveColor="default"
      size="large"
    >
      BLOG
    </Button>
  </Link>
  <Button
    activeColor="default"
    className="info-button"
    onClick={[Function]}
    passiveColor="default"
    size="large"
  >
    INFO
  </Button>
  <TitleImage
    className="footer-icon"
    link="https://github.com/AJarombek"
    src="./assets/github.png"
    title="GITHUB"
  />
</div>
`;

exports[`BlogPost No Sources matches snapshot 1`] = `
<div
  className="jarombek-blog-post"
>
  <div
    className="jarombek-blog-headers"
  >
    <p
      className="jarombek-blog-type"
    >
      DISCOVERY
    </p>
    <p
      className="jarombek-blog-date"
    >
      August 17th, 2018
    </p>
  </div>
  <Link
    className="jarombek-blog-title"
    replace={false}
    to="/blog/Test"
  >
    <p
      className="jarombek-blog-title-content"
    >
      Test
    </p>
  </Link>
  <div
    className="jarombek-blog-tags"
  >
    <TagList
      tagList={Array []}
    />
  </div>
  <div
    className="jarombek-blog-content"
  />
  <div
    className="jarombek-blog-sources"
  />
</div>
`;

exports[`BlogPost matches snapshot 1`] = `
<div
  className="jarombek-blog-post"
>
  <div
    className="jarombek-blog-headers"
  >
    <p
      className="jarombek-blog-type"
    >
      DISCOVERY
    </p>
    <p
      className="jarombek-blog-date"
    >
      August 17th, 2018
    </p>
  </div>
  <Link
    className="jarombek-blog-title"
    replace={false}
    to="/blog/Test"
  >
    <p
      className="jarombek-blog-title-content"
    >
      Test
    </p>
  </Link>
  <div
    className="jarombek-blog-tags"
  >
    <TagList
      tagList={Array []}
    />
  </div>
  <div
    className="jarombek-blog-content"
  />
  <div
    className="jarombek-blog-sources"
  >
    <p
      className="jarombek-blog-source"
      key="8efdedd8-b681-42ff-b921-44484d5ad4db"
    >
      [1] Start
      <a
        href="jarombek.com"
      >
        Link
      </a>
      End
    </p>
  </div>
</div>
`;

exports[`Button matches snapshot 1`] = `
<div>
  <button
    className="jarbek-button passive-color-default
                                active-color-default border-color-default
                                jarbek-button-large"
    onClick={[Function]}
  />
</div>
`;

exports[`Definition matches snapshot 1`] = `
<div
  className="jarombek-def"
>
  <p
    className="jarombek-def-word"
  >
    Snapshot Testing
  </p>
  <p
    className="jarombek-def-desc"
  >
     
  </p>
</div>
`;

exports[`Empty TagList matches snapshot 1`] = `
<div
  className="jarombek-tag-list"
>
  <p>
    No Tags
  </p>
</div>
`;

exports[`PictureButton matches snapshot 1`] = `
<div>
  <button
    className="jarbek-button active-color-default
                                passive-color-undefined jarbek-button-large"
  >
    <div
      className="jarbek-picture-button"
    >
      <div
        className="jarbek-pb-picture"
      >
        <img
          src={
            Object {
              "default": [Function],
            }
          }
        />
      </div>
      <div
        className="jarbek-pb-text"
      >
        Test
      </div>
    </div>
  </button>
</div>
`;

exports[`Tag matches snapshot 1`] = `
<div
  className="jarombek-tag jarombek-tag-large"
>
  <div
    className="jarombek-tag-color-javascript"
  >
    <figure>
      <img
        className="jarombek-tag-picture"
        src="https://asset.jarombek.com/jarombek.png"
      />
    </figure>
    <p
      className="jarombek-tag-name"
    >
      JavaScript
    </p>
  </div>
</div>
`;

exports[`TagList matches snapshot 1`] = `
<div
  className="jarombek-tag-list"
>
  <Tag
    color="javascript"
    id="js"
    key="JavaScript"
    name="JavaScript"
    picture=""
    showPicture={true}
  />
</div>
`;

exports[`TitleImage matches snapshot 1`] = `
<div>
  <figure
    className="jarbek-figure"
  >
    <img
      src="test-file-stub"
    />
  </figure>
  <p
    className="jarbek-image-title"
  >
    Test
  </p>
</div>
`;

exports[`TitleImage with Link matches snapshot 1`] = `
<div>
  <figure
    className="jarbek-figure"
  >
    <a
      href="jarombek.com"
    >
      <img
        src="test-file-stub"
      />
    </a>
  </figure>
  <p
    className="jarbek-image-title"
  >
    Test
  </p>
</div>
`;

exports[`WebsiteNav matches snapshot 1`] = `
<nav
  className="jarombek-nav"
>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    className="jarombek-nav-left"
    to="/blog"
  >
    BLOG
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    className="jarombek-nav-middle"
    to="/"
  >
    Andrew Jarombek
  </NavLink>
  <div
    className="jarombek-nav-right"
    onClick={[Function]}
  >
    <Button
      activeColor="primary"
      className="subscribe-button"
      onClick={[Function]}
      passiveColor="default"
      size="medium"
    >
      SUBSCRIBE
    </Button>
  </div>
</nav>
`;

exports[`WebsiteTemplate matches snapshot 1`] = `
<div
  className="jarombek-template"
>
  <WebsiteNav
    hideSubscribe={false}
    subscribeAction={[Function]}
  />
  <p>
    Hello
  </p>
</div>
`;
